### PR TITLE
Feature/reply ui

### PR DESCRIPTION
**Pull request by lee-benjamin**

### CODE REVIEW

File: src/js/components.js [hunk 1]
```diff
@@ -378,6 +421,59 @@ function NoTweets(props) {
   );
 }
 
+
+class Timeline extends React.Component {
+  constructor(props) {
+    super(props)
+    let tweetComponents = _.map(props.tweets, (tweet) =>
```
Comment (kev-chan) [inline src/js/components.js]: i'd recommend keeping all element creation in the render routine

Comment (lee-benjamin) [reply src/js/components.js]: done

File: src/styles/sass/style.scss [hunk 2]
```diff
@@ -157,54 +207,93 @@ $post-tweet-height: 75px;
...
+        .Tweet {
+          .Avatar {
+            margin-right: 20px;
+          }
+          width: $modal-width;
+        }
+        animation-duration: 1s;
+        animation-name: slide-in;
+        position: fixed;
+        top: 0px;
```
Comment (kev-chan) [inline src/styles/sass/style.scss]: top: 0;

Comment (lee-benjamin) [reply src/styles/sass/style.scss]: done

**Pull request merged**